File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/numar-sedinte-comisie-gl-report-filter.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";
import { DocumentIdentifierModel } from "./document-identifier.model ";

@JsonObject
export class NumarSedinteComisieGlReportFilterModel {

	@JsonProperty("dataSedintaDeLa", JsonDateConverter)
	public dataSedintaDeLa: Date = null;

	@JsonProperty("dataSedintaPanaLa", JsonDateConverter)
    public dataSedintaPanaLa: Date = null;
    
	@JsonProperty("comisieId", Number)
	public comisieId: number = null;
    
	@JsonProperty("document", DocumentIdentifierModel)
	public document: DocumentIdentifierModel = null;
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/NumarSedinteComisieGlReportFilterModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.Date;

public class NumarSedinteComisieGlReportFilterModel {

	private Date dataSedintaDeLa;
	private Date dataSedintaPanaLa;
	private Long comisieId;
	private DocumentIdentifierModel document;

	public Date getDataSedintaDeLa() {
		return dataSedintaDeLa;
	}

	public void setDataSedintaDeLa(Date dataSedintaDeLa) {
		this.dataSedintaDeLa = dataSedintaDeLa;
	}

	public Date getDataSedintaPanaLa() {
		return dataSedintaPanaLa;
	}

	public void setDataSedintaPanaLa(Date dataSedintaPanaLa) {
		this.dataSedintaPanaLa = dataSedintaPanaLa;
	}

	public Long getComisieId() {
		return comisieId;
	}

	public void setComisieId(Long comisieId) {
		this.comisieId = comisieId;
	}

	public DocumentIdentifierModel getDocument() {
		return document;
	}

	public void setDocument(DocumentIdentifierModel document) {
		this.document = document;
	}

}
